Rakefile.rb

```diff
@@ -27,8 +27,12 @@ def json_write(file, json)
   # we only build any country that contains that string. For example:
   #    EP_COUNTRY_REFRESH=Latvia be rake countries.json
   to_build = ENV['EP_COUNTRY_REFRESH'] || 'data'
-  countries = EveryPolitician.countries.select do |c|
-    c.slug.downcase.include? to_build.downcase
+  if to_build == 'data'
+    countries = EveryPolitician.countries
```

Comment:
@chrismytton I decided to take this even further, and extract all this logic out into a Task class instead, which made it significantly easier to refactor: https://github.com/everypolitician/everypolitician-data/pull/18484